ios/ExpoLiveActivityModule.swift

```diff
@@ -19,6 +19,9 @@ public class ExpoLiveActivityModule: Module {
     @Field
     var date: Double?
 
+    @Field
+    var progress: Double?
```

Comment:
I would change it to:
```
@Field
    var progressBar: ProgressBar?
      
    struct ProgressBar: Record {
      @Field
      var date: Double?

      @Field
      var progress: Double?
    }
```
And that way we don't have to map the JS object to that NativeState structure on the RN side.